RULES.md

```
@@ -150,6 +150,15 @@ it can be made safer by converting it into:
   #                   | Replace the expression with the environment variable
 ```
 
+## <a id="ADES105"></a> ADES105 - Expression in `addnab/docker-run-action` script
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: a

(MD033, no-inline-html)